doc/doxygen/notes_Doxy

```diff
@@ -0,0 +1 @@
+The "doxyhen.config.orig" was originaly in /engine directory and  all reletive  files ,edit to run from a directory other than }VS|/engine location (VSE_root)
```

Comment:
doxyhen => doxygen
originally in ./engine
relative
whitespace damage (multiple instances)